.travis.yml

```diff
@@ -1,13 +1,16 @@
 language: node_js
 node_js:
 - 8.11
+install:
+  - npm i -g npm@5.7.1 # this should be deleted once 5.7.X is out of "pre-release"
+  - npm ci             # instead of default npm install
 branches:
   only:
   - master
   - "/v\\d+\\.\\d+\\.d+$/"
 cache:
   directories:
-  - node_modules
+  - "$HOME/.npm"
```

Comment:
I'm assuming the node_modules folder is inside the `~/.npm/` folder?

Reply:
Yeah, and just caching `node_modules` gains us nothing when using ci, [per the docs](https://docs.npmjs.com/cli/ci):

`If a node_modules is already present, it will be automatically removed before npm ci begins its install.`

Reply:
If this is caching the `node_modules` folder, I'm not sure it's worth it. I just noticed this comment on the NPM docs page: "If a node_modules is already present, it will be automatically removed before npm ci begins its install." (https://docs.npmjs.com/cli/ci#description)

Reply:
Oh, nevermind, I just looked it up. I guess the `~/.npm` folder is a cache, not the actual `node_modules` folder. So, I think this makes sense :+1:.